src/abTestingInfra/tests/utils.spec.ts

```diff
@@ -0,0 +1,25 @@
+import assert from "assert";
+import { describe, it, beforeEach, afterEach } from "mocha";
+
+import { isValidName } from "../utils";
```

Comment:
Same q as above — would colocation in the same dir work?

Reply:
Fixed